azext_iot/_params.py

```diff
@@ -819,6 +819,13 @@ def load_arguments(self, _):
             type=int,
             help="Maximum number of configurations to return. By default all configurations are returned.",
         )
+        context.argument(
+            "custom_metric_queries",
+            options_list=["--custom-metric-queries"],
+            type=dict,
+            help="Map of custrom metric queries for configuration. "
+            'Format example: {"mertic1":"select *", "metric2":"select moduleId from devices.modules where tags.location=''US''"}',
```

Comment:
usually we have format examples in the help rather than the param help.

Reply:
ok got it, I was following the one above, will remove that